tools/topology/topology1/sof-mt8195-mt6359-rt1019-rt5682.m4

```diff
@@ -53,7 +53,15 @@ define(matrix2, `ROUTE_MATRIX(3,
 
 ifdef(`GOOGLE_RTC_AUDIO_PROCESSING', `MUXDEMUX_CONFIG(demux_priv_1, 2, LIST_NONEWLINE(`', `matrix1,', `matrix2'))')
 ifdef(`GOOGLE_RTC_AUDIO_PROCESSING', `define(`SPK_PERIOD_US', 10000)', `define(`SPK_PERIOD_US', 1000)')
-ifdef(`GOOGLE_RTC_AUDIO_PROCESSING', `define(`MIC_PERIOD_US', 10000)', `define(`MIC_PERIOD_US', 2000)')
+
+ifdef(`GOOGLE_RTC_AUDIO_PROCESSING',
+	`define(`MIC_PERIOD_US', 10000)'
+	,
+	`ifdef(`RTNR',
+# 5ms period is required for RTNR build 20220728 and later versions
```

Comment:
@MingJenTai  Just wonder how to check "RTNR" for 20220728 and later? Could you please explain it? Thanks.

Reply:
Previously we provided the required commits and library version information to the system integrator( or the one who will do the integration ).